unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Tests/BuilderProjectsPanelControllerShould.cs

```diff
@@ -40,7 +41,79 @@ public void SetUp()
         }
 
         [TearDown]
-        public void TearDown() { controller.Dispose(); }
+        public void TearDown()
+        {
+            controller.OnJumpInOrEdit -= AssertJump;
+
+            controller.Dispose();
+        }
+
+        [Test]
+        public void LandsFetchedError()
+        {
+            //Arrange
+            controller.isFetching = true;
+            var error =  "TestError";
+
+            //Act
+            controller.LandsFetchedError(error);
+
+            //Assert
+            Assert.IsFalse(controller.isFetching);
+            LogAssert.Expect(LogType.Error, error);
+        }
+
+        [Test]
+        public void LandsFetched()
+        {
+            //Arrange
+            Parcel parcel = new Parcel();
+            parcel.x = 0;
+            parcel.y = 0;
+
+            Vector2Int parcelCoords = new Vector2Int(0, 0);
+            Land land = new Land();
+            land.parcels = new List<Parcel>() { parcel };
+
+            LandWithAccess landWithAcces = new LandWithAccess(land);
+            DeployedScene deployedScene = new DeployedScene();
+            deployedScene.parcelsCoord = new Vector2Int[] { parcelCoords };
+            deployedScene.deploymentSource = DeployedScene.Source.SDK;
+
+            landWithAcces.scenes = new List<DeployedScene>() { deployedScene };
+            var lands = new LandWithAccess[]
+            {
+                landWithAcces
+            };
+
+            //Act
+            controller.LandsFetched(lands);
+
+            //Assert
+            landsController.Received().SetLands(lands);
+        }
+
+        [Test]
+        public void GoToCoords()
+        {
+            //Arrange
+            controller.OnJumpInOrEdit += AssertJump;
+
+            //Act
+            controller.GoToCoords(new Vector2Int(0, 0));
+        }
+
+        [Test]
+        public void OnGoToEditScene()
+        {
+            //Arrange
+            controller.OnJumpInOrEdit += AssertJump;
+
+            //Act
+            controller.OnGoToEditScene(new Vector2Int(0, 0));
+        }
```

Comment:
You missed some assertions here.

Reply:
The assertions are in the previous event  `private void AssertJump() { Assert.Pass(); }`

Reply:
but `Assert.Pass()` makes the test pass, when does this test fails?

Reply:
Added the Assert part